### ISSUE TITLE
Use "G29 A1" instead of "G29 1" to just probe the Bed area where the prints going to be

### ISSUE THREAD

**Issue by vordenken** [bambulab/BambuStudio#5472]
**Is your feature request related to a problem? Please describe.**

Probing the whole bed before every print takes a lot of time (but probing in general is ofc really good).

**Describe the solution you'd like**

The probing should only be done in the area where the print is going to be.
Heres a video shown this exact solution: https://www.youtube.com/watch?v=Iqyv5C6SLIk

**Comment by Anthony-Bec** [comment]
Yes! That would be such a timesaver and its super easy to change. I better see BambuLab change that!
